### PR TITLE
AWS trace wrapper reduce message attributes

### PR DESCRIPTION
**Pull request by stroem**
Will update will only inject the datadog trace headers.

### CODE REVIEW

File: v2/trace/http-headers_test.go [hunk 1]
```diff
@@ -9,6 +9,13 @@ import (
 	"github.com/SKF/go-utility/v2/trace"
 )
 
+func Test_AllHeaders(t *testing.T) {
+	const expectedLength = 3 + 5
```
Comment (lonnblad) [inline v2/trace/http-headers_test.go:13]: should we add constants in the test like:
```
const noOfB3Headers = 3
const noOfDatadogHeaders = 5
```

Comment (stroem) [reply v2/trace/http-headers_test.go]: I agree. Updated

File: v2/trace/aws/inject.go [hunk 2]
```diff
@@ -77,18 +77,16 @@ func getTraceAttributesFromContext(ctx context.Context) map[string]string {
 	if span := oc_trace.FromContext(ctx); span != nil {
 		spanCtx := span.SpanContext()
 
-		attributes[b3TraceHeader] = hex.EncodeToString(spanCtx.TraceID[:])
```
Comment (j0s) [inline v2/trace/aws/inject.go:80]: Should we no longer provide B3 headers?

Comment (stroem) [reply v2/trace/aws/inject.go:80]: We are trying to minimize the amount of message attributes in use (due to the limitation of max 10 messages attributes for SQS). Going from 4 to 2.
The aws wrapper still converts the b3 headers to dd, and the only subscriber/receiver of these headers we have currently is datadog. We don't have any lambdas currently running opencensus (what I know of), if that is something we want in the future, we need to add a subscriber/reciever for creating opencensus spans and maybe re-add this header (and maybe rewrite this solution to combine all headers in just one message attribute, e.g json).

Comment (j0s) [reply v2/trace/aws/inject.go:80]: Ok!

**Review by j0s — APPROVED** [review]

**Pull request merged**